platform/darwin/src/MGLRendererConfiguration.mm

```diff
@@ -41,9 +68,7 @@ - (const float)scaleFactor {
 }
 
 - (BOOL)perSourceCollisions {
-    NSNumber *boolWrapper = [[[NSBundle mainBundle] infoDictionary] valueForKey:@"MGLCollisionBehaviorPre4_0"];
-    
-    return boolWrapper.boolValue;
+    return [[NSUserDefaults standardUserDefaults] boolForKey:MGLCollisionBehaviorPre4_0Key];
```

Comment:
Yes, originally I had it (essentially) like that, but changed to moving the logic to the initializers for testing purposes.

Reply:
There's also something about having the accessory only depend on `NSUserDefaults` that I like, though perhaps this should go even further and move **all** the logic into the initializer, leaving `perSourceCollisions` as a simple ivar backed property.

This would work currently since `-[MGLRendererConfiguration currentConfiguration]` returns a new instance with each invocation. (*An aside: since `currentConfiguration` doesn't do anything I'm tempted to remove it too*).

Reply:
Rejigged in recent push.